locales/zh_CN/LC_MESSAGES/sphinx.po

```diff
@@ -93,13 +93,13 @@ msgstr "本地化"
 
 #: ../../_templates/layout.html:45
```

Comment:
That file actually should not exist, I think we rejected it, and only keep it on our minds, should be find something else to use it for in the future.

Reply:
`_templates/layout.html` is actually a built-in Sphinx template. Here is the overwrite content, which can be modified later.

Reply:
But why have it, it would block their updates when I update Sphinx while not serving a purpose right now.